localstripe/resources.py

```diff
@@ -244,6 +244,14 @@ def do_expand(path, obj):
         return obj
 
 
+class DeletedObject(StripeObject):
+    deleted = True
+
+    def __init__(self, id, object):
+        self.id = id
+        self.object = object
```

Comment:
An optimization could make it simpler to call the class:
```suggestion
    def __init__(self, deleted_object):
        self.id = deleted_object.id
        self.object = deleted_object.object
```

(take it or leave it)

Reply:
we don't have the `deleted_object` in `_api_delete`, we only have its `id` and its class, so doing this would require to fetch the object first like this:

```python
deleted_obj = store.get(cls.object + ':' + id)
...
del store[key]
return DeletedObject(deleted_obj)
```

which seems a bit heavy just to save an argument, so I prefer keeping it as is!

Reply:
You're right :+1: